diagnostics-extension/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "diagnostics-extension",
+  "version": "1.0.0",
+  "description": "Extension for Sample Diagnostics App",
+  "main": "index.js",
+  "scripts": {
+    "prebuild": "sudo rm -rf build/",
+    "build": "webpack",
+    "postbuild": "cp -R public/** build",
```

Comment:
When `npm run build` is run, the `pre` and `post` automatically run before and after the build command respectively.